src/app/utility/upload-data/upload-data.component.css

```diff
@@ -0,0 +1,7 @@
+.import-meter .file-name-width{
```

Comment:
Duplicated in other css files. Move to global.css

Reply:
.import-meter .file-name is used in multiple places but isn't consistent in all the places. Going to leave there